fabric/fabric-utils/src/test/java/io/fabric8/utils/FabricValidationsTest.java

```diff
@@ -102,6 +108,25 @@ public void uris() {
         assertTrue(isURIValid("http://host.name:8181/path"));
         assertTrue(isURIValid("http://u@host.name:8181/path"));
         assertTrue(isURIValid("http://u:p@host.name:8181/path"));
+        // ENTESB-7303
+        assertTrue(isURIValid(new File("/tmp/x").toURI().toString()));
+        // authority (user:password@host:port) == null, path == //tmp/x
+        assertTrue(isURIValid("file:////tmp/x"));
```

Comment:
it might be worthy to add some tests with windows path using file:/C:/tmp/x to file:///C:/tmp/x
(Talking by bad experience, we are currently struggling with this kind of issue with bugs in Ivy/Groovy impacting Camel/Fuse Tooling)

Reply:
Thanks - it's worth considering. Let me check